library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -233,6 +254,7 @@ sealed class Conversation {
         return when (this) {
             is V1 -> conversationV1.decryptedMessages(limit, before, after, direction)
             is V2 -> conversationV2.decryptedMessages(limit, before, after, direction)
+            is Group -> TODO()
```

Comment:
The decryptedMessage stuff is needed for React Native to be able to support custom content types.